tests/CMakeLists.txt

```diff
@@ -37,4 +37,5 @@ endif()
 include_directories(${GTEST_INCLUDE_DIRS})
 target_link_libraries(dummy_test ${GTEST_BOTH_LIBRARIES})
 
-add_subdirectory(carddatabase)
\ No newline at end of file
+add_subdirectory(carddatabase)
+add_subdirectory(loading_from_clipboard)
```

Comment:
I'd like to have a separate top-level `test/cockatrice/foo.cpp` and `test/common/bar.cpp` rather than `project/tests` just so it's easier to keep the code isolated